### ISSUE TITLE
Font-size widget ugly when initializing TinyMCE inside of "display: none".

### ISSUE THREAD

**Issue by drumsibumsi** [tinymce/tinymce#4018]
This is bug report.

1. Configure TinyMCE to show font size on the toolbar.
2. Initialize TinyMCE inside the "display: none" container.
3. After a while make the container visible ("display: block").
4. Note that everything is correct, EXCEPT font-size widget.

Here is the DEMO (Classic Example of TinyMCE):
https://codepen.io/tinymce/pen/YybwmL

1. I put the HTML inside
`<div id="div" style="display: none;">`
2. I put
```
window.setTimeout(function() {
  $("#div").show();
 }, 2000);
```
in the end of JavaScript.

![screenshot-codepen io-2017-10-23-16-14-50](https://user-images.githubusercontent.com/2128661/31890951-9526d0e2-b80d-11e7-89cc-f3be00410d6f.png)

Previous versions before 4.7.X didn't have this issue.

**Comment by jayarjo** [comment]
Cannot replicate it. I've forked your pen here: https://codepen.io/anon/pen/LzKoQQ. What is the browser where it happens?

**Comment by drumsibumsi** [comment]
Indeed.

The example fails on Firefox 56.0.1 (64-bit) (latest) on Windows 10.
Works correctly on Chrome (latest).
Works correctly on Edge (latest).
Works correctly on Internet Explorer 11.

Not using plugins on any browsers.

**Comment by spocke** [comment]
Yes, this happens on firefox only. That browser can't compute css on something that is hidden so we need to fix this in core. Thanks for the details on this one.

**Comment by drumsibumsi** [comment]
Seems to be fixed in 4.7.2, although not mentioned in changelog.
